obspy/io/arclink/inventory.py

```diff
@@ -0,0 +1,908 @@
+        decimation_factor = _attr2obj(stage, "decimationFactor", int)
+        if rate != 0.0:
+            temp = _attr2obj(stage, "delay", float) / rate
+            decimation_delay = _read_float_var(temp,
+                                               FloatWithUncertaintiesAndUnit,
+                                               unit=True)
+            temp = _attr2obj(stage, "correction", float) / rate
+            decimation_corr = _read_float_var(temp,
+                                              FloatWithUncertaintiesAndUnit,
+                                              unit=True)
+        else:
+            decimation_delay = _read_float_var("inf",
+                                               FloatWithUncertaintiesAndUnit,
+                                               unit=True)
+            decimation_corr = _read_float_var("inf",
+                                              FloatWithUncertaintiesAndUnit,
+                                              unit=True)
+        decimation_input_sample_rate = \
+            _read_float_var(rate, Frequency)
+        decimation_offset = int(0)
+    elif(elem_type == "datalogger"):
+        decimation_factor = int(1)
+        decimation_delay = _read_float_var(0.00,
+                                           FloatWithUncertaintiesAndUnit,
+                                           unit=True)
+        decimation_corr = _read_float_var(0.00,
+                                          FloatWithUncertaintiesAndUnit,
+                                          unit=True)
+        decimation_input_sample_rate = \
+            _read_float_var(rate, Frequency)
+        decimation_offset = int(0)
+    elif(elem_type == "responsePAZ" or elem_type == "responsePolynomial"):
+        decimation_factor = None
+        decimation_delay = None
+        decimation_corr = None
+        decimation_input_sample_rate = None
+        decimation_offset = None
+    else:
+        raise ValueError("Unknown type of response: " + str(elem_type))
+
+    # set up list of for this stage arguments
+    kwargs = {
+        "stage_sequence_number": stage_sequence_number,
+        "input_units": str(input_units),
+        "output_units": str(output_units),
+        "input_units_description": None,
+        "output_units_description": None,
+        "resource_id": None,
+        "resource_id2": resource_id,
+        "stage_gain": stage_gain,
+        "stage_gain_frequency": stage_gain_frequency,
+        "name": name,
+        "description": None,
+        "decimation_input_sample_rate": decimation_input_sample_rate,
+        "decimation_factor": decimation_factor,
+        "decimation_offset": decimation_offset,
+        "decimation_delay": decimation_delay,
+        "decimation_correction": decimation_corr
+    }
+
+    # Different processing for different types of responses
+    # currently supported:
+    # PAZ
+    # COEFF
+    # FIR
+    # Polynomial response is not supported, could not find example
```

Comment:
Seems like it's implemented below alright..

Also, I think the `if/elif` clause below could use an..

```python
else:
    msg = "Unsupported response type (%s). Please contact developers." % elem_type
    warnings.warn(msg)
```